v3/lint/registration_test.go

```diff
@@ -152,6 +162,116 @@ func TestRegister(t *testing.T) {
 	}
 }
 
+func TestRegistryWithRevocationListLitns(t *testing.T) {
```

Comment:
optional: This doesn't seem to follow the rest of the tests in this file. It isn't clear to me from the title exactly what functionality is being tested. Maybe we can give it a better name?